api/lib/services/csvwBuilder.ts

```diff
@@ -103,7 +104,16 @@ export function buildCsvw (tableOrDataset: DataCube.Table | DataCube.DimensionTa
     return mapped
   }, [] as Csvw.Column[])
 
-  csvwGraph.tableSchema.columns = [...mappedColumns, ...suppressedColumns]
+  const additionalColumns: Csvw.Column[] = []
+  if (table.types.has(dataCube.FactTable)) {
```

Comment:
By the way. I notice that there is now no test which sets up `dataCube.FactTable`?

Reply:
Technically some of the tests using fixtures (specGraph) were already setting `FactTable`, but I added a test specifically to test this new virtual column.